swarm/network/protocol_test.go

```diff
@@ -71,19 +74,36 @@ func HandshakeMsgExchange(lhs, rhs *HandshakeMsg, id enode.ID) []p2ptest.Exchang
 }
 
 func newBzzBaseTester(t *testing.T, n int, prvkey *ecdsa.PrivateKey, spec *protocols.Spec, run func(*BzzPeer) error) (*bzzTester, error) {
```

Comment:
I'm not sure what `n int` means. I see that we generate that many addresses. What does that mean? Is an address represent a stripped down node?

Reply:
they represent a peer connection so stripped  down  that all you  can do is send and receive protocol. messages (trigger//expect)